### PR TITLE
Portfolio: Desktop Version

### PR DESCRIPTION
**Pull request by MoyasiGinko**
### In this milestone project, I:

- created a new GitHub repo `Portfolio-Site`.
- created a new branch `desktop-ver-site`.
- created a `.gitignore` file.
- added `node_modules/` to `.gitignore` file.
- created `.github/workflows/linters.yml `file.
- added `license.md` file and modified the `README.md` file.
- added dependencies for html & css.
- updated html file and css file.
- modified social icons in `index.html` file.
- added `media-queries` for desktop version.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -486,12 +582,28 @@ header {
     margin-left: 0;
     margin-top: 10px;
   }
+
+  #contact-container {
+    padding: 12% 30%;
+    background-image: url(./asset/contact-form-bg-desktop.svg);
+    background-size: 94%;
+    background-position: 60% 30%;
+  }
```
Comment (Zuhaib042) [inline style.css:591]: - [ ] All the elements in the contact section are centered on the page including the button in the figma design but in your design, the button is not centered on the desktop screen as it should be, so kindly consider fixing it to make it identical to the figma design.
your design
vc

figma design
f

File: index.html [hunk 2]
```diff
@@ -11,64 +11,66 @@
...
+                                    </li>
+                                </ul>
+                            </div>
+                    </div>
                 </div>
-            </div>
-        </section>
+            </section>
+        </div>
         <section id="card-section">
```
Comment (Zuhaib042) [inline index.html:74]: - [ ] In the cards container, half of it contains the description, and half of it contains the picture, and in each card this discipline switch places (in first card, the picture on the left and the description on the right, in the next card the description on the left and the picture on the right and so on and so forth), but in your design, card no 3 and 4 contradict this rule so kindly consider fixing it.
your design
re

figma design
p

File: index.html [hunk 3]
```diff
@@ -11,64 +11,66 @@
...
             <div class="card__container">
                 <div class="items card-1">
```
Comment (Zuhaib042) [inline index.html:76]: - [ ] In your design the margin between all the description elements is less than what it should be to make it identical to the figma design. One of the main requirements is to make our design's fonts, margins and paddings identical to the figma design so kindly consider measuring the margin between those elements in the figma design and then apply that in your design.
- [ ] Also the margin between the `See Project` button and the languages is none so kindly take extra care to fix that.
your design
as

figma design
qw

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @MoyasiGinko  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Professional PR
✅ Overall Good Styling and Design

## Required Changes ♻️
- [ ] Kindly consider changing this styling `grid-template-columns: 45% 55%;` on line 534 in style.css to `grid-template-columns: 40% 60%;` to make it identical to the figma design. I am writing this here instead of inline-comment because you haven't introduced that code in this PR so I couldn't do inline-comment.
- [ ] Kindly consider changing the `language, frameworks, and skills` headings' color to `#344563` to make it identical to the figma design as it is a project requirement.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (MoyasiGinko) [reply index.html:76]: I have fixed the issues on the `see project` button.

**Review by huseyinbiyik — APPROVED** [review]
Hi @MoyasiGinko 

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions
- It seems you need a gray background and the correct alignment for each programming language here. I think it is better to stick to the design and implement it. 👍 
Screenshot 2023-04-08 at 12 07 05 AM
Screenshot 2023-04-08 at 12 07 13 AM

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**